src/Uno.UI/DataBinding/BindingPropertyHelper.cs

```diff
@@ -333,7 +333,7 @@ private static bool InternalIsEvent(Type type, string property)
 						return attachedPropertyGetter.ReturnType;
 					}
 
-					if (type.IsPrimitive && property == "Value")
+					if (type.IsValueType && property == "Value")
```

Comment:
This won't interfere with strings?

Reply:
Ok I see... I thought `string` were considered as primitives... it's not.  So that's correct.
https://docs.microsoft.com/en-us/dotnet/api/system.type.isprimitive?view=net-6.0#remarks